packages/ui/src/constants/abis.ts

```diff
@@ -0,0 +1,314 @@
+export const erc1155ABI = [
```

Comment:
I'm surprised viem doesn't return this abi. Also where is the one for erc721?

Reply:
Yeah kind of annoying, viem/wagmi doesn't have an abi for erc1155: [https://wagmi.sh/react/constants/abis](https://wagmi.sh/react/constants/abis)